test/test_helper.rb

```diff
@@ -3,7 +3,6 @@
 ENV['RACK_ENV'] = 'test'
 require 'simplecov' if ENV['COVERAGE']
 
-gem 'minitest', '=5.4.1'
```

Comment:
This was required because Activesupport 4.2.1 has bumped minitest version to 5.5.1